src/components/Modal/docs/index.story.tsx

```diff
@@ -0,0 +1,100 @@
+import * as React from 'react';
+import * as examples from './examples';
+import {
+  header,
+  api,
+  description,
+  divider,
+  playground,
+  tab,
+  code as baseCode,
+  tabs,
+  testkit,
+  title,
+} from 'wix-storybook-utils/Sections';
+import { allComponents } from '../../../../stories/utils/allComponents';
+import { settingsPanel } from '../../../../stories/utils/SettingsPanel';
+import * as ModalWiringExampleRaw from '!raw-loader!./ModalWiringExample.tsx';
+import * as ModalWiringExampleCSSRaw from '!raw-loader!./ModalWiringExample.st.css';
+import { ModalWiringExample } from './ModalWiringExample';
+import { Modal } from '../';
+import * as Readme from '../../Modal/README.md';
+
+const code = config =>
+  baseCode({ components: allComponents, compact: true, ...config });
+
+export default {
+  category: 'Components',
+  storyName: 'Modal',
+  component: Modal,
+  componentPath: '../Modal.tsx',
+  componentProps: () => ({
+    'data-hook': 'storybook-Modal',
+  }),
+  exampleProps: {
+    isOpen: false,
+    inFullScreen: false,
+    withCloseButton: false,
+    withBackground: false,
+    onRequestClose: () => {
+      console.log('on request close');
+    },
```

Comment:
The problem with the `exampleProps` is that once the Modal is opened in the Playground, there's no way to close it.
Maybe we can use `componentWrapper` in order to add a state to the example.
You can see the documentation here:
https://github.com/wix/wix-ui/blob/aca805fe29ba11be17c85f492af310681dda7b86/packages/wix-storybook-utils/docs/usage.md (under `componentWrapper`)